src/components/cc-addon-linked-apps/cc-addon-linked-apps.js

```diff
@@ -112,10 +111,10 @@ export class CcAddonLinkedApps extends LitElement {
         }
 
         .details {
-          --cc-align-items: center;
-          --cc-gap: 0.5em;
-
+          display: flex;
+          flex-wrap: wrap;
```

Comment:
**💣 chore:** ...but I'd place it there. WDYT? (2/2)

```suggestion
          flex-wrap: wrap;
          align-items: center;
```

Note: actually, it was previously here with `--cc-align-items: center;`.

Reply:
You are right! I broke `cc-zone` with my previous change. I'll go for your solution (i'll edit the commit directly instead of creating a new one :+1:)

Thanks @roberttran-cc :rocket: